### PR TITLE
Add link to rescue dashboard in dmo portal

### PR DESCRIPTION
**Pull request by aswins**
Adds link to https://process.keralarescue.in from DMO portal

### CODE REVIEW

File: templates/dmodash.html [hunk 1]
```diff
@@ -6,9 +6,12 @@
...
 <div class="container text-center">
     <h1 class="main-logo">kerala<span style="font-weight: normal;">rescue  DMO Portal</span></h1>
-    <p style="margin-bottom:20px;">
-        <i>Please Select the district to view stats </i><br/>
-  </p>
+        <p style="margin-bottom:20px;">
+            <a href="https://process.keralarescue.in/"><b>Click here to view the rescue dashboard</b></a><br/>
+        </p>
+        <p style="margin-bottom:20px;">
+            <i>Please Select the district to view stats </i><br/>
```
Comment (shidhincr) [inline templates/dmodash.html]: `view stats`
@aswins Is this a typo ?

Comment (aswins) [reply templates/dmodash.html]: Nope. That shows stats. Also that was existing code. Just indented.

Comment (aswins) [reply templates/dmodash.html]: Done

**Comment by laxman954** [comment]
@aswins please resolve the conflict

**Pull request closed without merging**